CombinatorialGames/Game/Specific.lean

```diff
@@ -0,0 +1,130 @@
+/-
+Copyright (c) 2022 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios, Tristan Figueroa Reid
+-/
+import CombinatorialGames.Game.PGame
+
+/-!
+# Specific games
```

Comment:
I think 'Specific' is a bad word here, because other 'specific' games defined in the literature are sometimes modifiers on games (i.e. Tiny & Miny games): perhaps 'Special' is a better word, since these types of games usually have interesting resulting theorems?

Reply:
(Or, we can have a separate file for those special games/game modifiers)

Reply:
I wasn't aware that "specific" games were already a thing! I think we can keep calling them "special", then.